app/models/user.rb

```diff
@@ -83,8 +83,8 @@ def self.to_csv
   scope :unconfirmed, -> { where(confirmed_at: nil) }
   scope :locked_out, -> { where.not(locked_at: nil) }
 
-  # @return [Integer] Number of users who have added at least one note, given the note body is not empty
-  scope :has_notes, -> { joins(:notes).distinct.count { |user| user.notes.any? { |note| !note.body_is_empty } } }
+  # Users who have added at least one note, given the note body is present
```

Comment:
What would be a good method name for the new instance predicate? Perhaps `has_notes?`?

Reply:
[cf0bab0](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/622/commits/cf0bab079b38fac28eb90ad2ca7adef0ecfb7757)